docs/pages/User/dashboard.md

```diff
@@ -44,6 +44,44 @@ https://localhost:6443
 ```
 **NOTE:** The PORT exposed with the port forward, that in the example is ``6443``, can be any PORT that is not already used.
 
+### Ingress
+LiqoDash can also be exposed using an [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress).
+You must have an [Ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/)
+for an Ingress resource to work. If you have one, you can specify an host exporting the variable `DASHBOARD_INGRESS` **before**
+installing Liqo, for example:
+```
+export DASHBOARD_INGRESS=example.liqodash.com
+```
+Once installed, you can see that an Ingress resource is created, accessible through:
+```
+kubectl get -n liqo ingress
+```
+Along with a secret that you will have to edit, providing a certificate and key. You can do it
+with the following command:
```

Comment:
AFAIK, the secret is automatically generated with a self-signed certificate. Hence, no actions are required it this is the behavior desired. Conversely, to have a valid certificate it is either possible to edit that secret (something along the lines of `kubectl replace secret tls ${CERT_NAME} --key ${KEY_FILE} --cert ${CERT_FILE}`) or use ad-hoc tools such as `cert-manager`, which nonetheless require to add an annotation to the ingress. In the latter case, I am not sure what would it happen after an upgrade of Liqo (is the additional annotation deleted by helm?)

Reply:
My 2 cents: IIRC I think that the secret is not automatically generated. If nginx does not find the secret, it automatically switch to the default one.

Reply:
@palexster Thanks, you are definitely right. I just checked and this is also the current behavior of nginx. However, I feel there is still no need to manually create the secret unless a custom certificate is to be provided. What do you think?

Reply:
Totally agree.